drivers/d3d12/SCsub

```diff
@@ -41,6 +41,12 @@ if env["use_pix"]:
     env_d3d12_rdd.Append(CPPEXTPATH=[env["pix_path"] + "/Include"])
 
 
+# Direct composition.
+
+if "dcomp" in env.get("supported", []):
+    env_d3d12_rdd.Append(CPPDEFINES=["DCOMP_ENABLED"])
```

Comment:
This isn't enough, as it's used in `rendering_context_driver_d3d12.h` which is included in platform code outside `env_d3d12_rdd`. So this would need to be set on the main `env` too.

Reply:
- #106827.